.github/workflows/well-known-environment.yml

```diff
@@ -145,7 +145,7 @@ jobs:
       - name: Configure AWS credentials for account
         uses: ./.github/actions/configure-aws-credentials
         with:
-          environment-name: ${{ env.branch }}
+          environment-name: $TARGET_ACCOUNT_NAME
```

Comment:
@A-Ashiq are you sure?  I'm not sure where this comes from?

Reply:
@kevinkuszyk it is already in use in previous jobs, further up this file. And it gets set in the `well-known-environment-name` action